addons/langpacks.rst

```diff
@@ -62,12 +62,17 @@ Langpacks: How To…
 How to handle New Languages for Release
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Context: the underlying issue for this lies within `issue-15353`_ and is on the roadmap to be fixed by the AMO team.
-Some related work is happening in H2 2021.
-
+Context: the underlying issue for this lies within `issue-15353`_ and can be
+fixed in addonscript by switching to the v5 AMO API for submission.  In the
```

Comment:
Note that it's not just a version change, the new API also has a subtly different path, FYI.

Reply:
Thanks, added a pointer to the corresponding jira issue which has more details on it.